Cargo.toml

```diff
@@ -37,11 +41,17 @@ tokio = { version = "1.15", optional = true, features = ["rt"] }
 
 [features]
 default = []
+dox = ["gtk/dox", "adw/dox"]
 libadwaita = ["adw"]
 libpanel = ["panel"]
```

Comment:
I think we should add libpanel to the dox feature too if possible.

Reply:
I missed that. I didn't know the dependency was also gtk related.